### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by snayrouz**
This branch has updated styling for edit forms and buttons.

@mimilettd 
@amhursh

### CODE REVIEW

File: app/controllers/bike_share_app.rb [hunk 1]
```diff
@@ -25,6 +30,13 @@ class BikeShareApp < Sinatra::Base
   end
 #weather conditions
   get '/conditions' do
+    Condition.connection
+    @pages = Condition.paginate(page: params[:page])
+    @conditions = Condition.order('date DESC').page(params[:page])
```
Comment (amhursh) [inline app/controllers/bike_share_app.rb:35]: This is cool stuff. Great work.

File: app/controllers/bike_share_app.rb [hunk 2]
```diff
@@ -25,6 +30,13 @@ class BikeShareApp < Sinatra::Base
...
+    erb :"conditions/index"
+  end
+
+  get '/conditions' do
```
Comment (mimizotti) [inline app/controllers/bike_share_app.rb:39]: @snayrouz Did you test this?

File: app/controllers/bike_share_app.rb [hunk 3]
```diff
@@ -58,6 +70,17 @@ class BikeShareApp < Sinatra::Base
...
+    erb :'conditions/dashboard'
+  end
+
+  get '/trips' do
+    Trip.connection
+    @pages = Trip.paginate(page: params[:page])
+    @trips = Trip.order('start_date DESC').page(params[:page])
+    erb :"trips/index"
+  end
+
```
Comment (mimizotti) [inline app/controllers/bike_share_app.rb:83]: Looks good. Is it a given that paginate will only display the first 30 trips?

File: app/models/condition.rb [hunk 4]
```diff
@@ -1,3 +1,11 @@
+require 'will_paginate'
+require 'will_paginate/active_record'
+
 class Condition < ActiveRecord::Base
 acts_as_copy_target #for postgres copy
+
+  def self.per_page
+    30
+  end
+
```
Comment (mimizotti) [inline app/models/condition.rb:10]: Nevermind. This answered my previous question.

File: app/views/conditions/edit.erb [hunk 5]
```diff
@@ -1,23 +1,39 @@
...
+    </div>
+    <div class="col-md-6 mb-1">
+      <label for="validationCustom01">Mean Humidity: </label>
+      <input type="text" class="form-control" id="validationCustom01" placeholder="condition[mean_humidity]" value="<%= @condition.mean_humidity %>" required>
+    </div>
+    <div class="col-md-6 mb-1">
+      <label for="validationCustom01">Mean Visibility: </label>
+      <input type="text" class="form-control" id="validationCustom01" placeholder="condition[mean_visibility]" value="<%= @condition.mean_visibility %>" required>
+    </div>
+    <div class="col-md-6 mb-1">
```
Comment (amhursh) [inline app/views/conditions/edit.erb:29]: nice and clean. looks good

File: app/public/vendor/bootstrap/css/bootstrap.css [hunk 6]
```diff
@@ -6,7 +6,7 @@
 
 /*! normalize.css v3.0.0 | MIT License | git.io/normalize */
 html {
-  font-family: sans-serif;
+  font-family: serif;
   -webkit-text-size-adjust: 100%;
       -ms-text-size-adjust: 100%;
 }
```
Comment (mimizotti) [inline app/public/vendor/bootstrap/css/bootstrap.css:12]: Do we want to use `serif` as the universal font for our website?

File: app/views/conditions/edit.erb [hunk 7]
```diff
@@ -1,23 +1,39 @@
...
+      <label for="validationCustom01">Mean Windspeed: </label>
+      <input type="text" class="form-control" id="validationCustom01" placeholder="condition[mean_windspeed]" value="<%= @condition.mean_windspeed %>" required>
+    </div>
+    <div class="col-md-6 mb-1">
+      <label for="validationCustom01">Precipitation: </label>
```
Comment (amhursh) [inline app/views/conditions/edit.erb:34]: Validation. awesome add.

File: app/views/conditions/new.erb [hunk 8]
```diff
@@ -1,7 +1,7 @@
 
 <h1>Add a new weather condition:</h1>
 
-<form action="/conditions" method="post">
+<form action="/conditions" method="POST">
```
Comment (amhursh) [inline app/views/conditions/new.erb:4]: good style catch

File: app/public/vendor/bootstrap/css/bootstrap.css [hunk 9]
```diff
@@ -77,6 +77,15 @@ sup {
   line-height: 0;
   vertical-align: baseline;
 }
+
+.card{
+  border: 1px solid red;
```
Comment (mimizotti) [inline app/public/vendor/bootstrap/css/bootstrap.css:82]: Where is this coming from?

File: app/public/vendor/bootstrap/css/bootstrap.css [hunk 10]
```diff
@@ -77,6 +77,15 @@ sup {
...
+}
+
+.card-title {
+  color: red;
```
Comment (mimizotti) [inline app/public/vendor/bootstrap/css/bootstrap.css:86]: Same as above. I didn't notice any red card on our website.

File: app/views/stations/index.erb [hunk 11]
```diff
@@ -1,10 +1,19 @@
-<h1>All Stations<a href="/station-dashboard"> Station Dashboard</a></h1>
-
+<h1>All Stations</h1>
 <div>
-  <% @stations.each do |station| %>
-    <h2><a href="/stations/<%= station.id %>"><%= station.name %></a></h2>
+  <a class="btn btn-primary" href="/station-dashboard"> Station Analytics</a>
```
Comment (mimizotti) [inline app/views/stations/index.erb:3]: Extra space between > and Station. Not sure if that matters.

File: app/views/stations/index.erb [hunk 12]
```diff
@@ -1,10 +1,19 @@
...
+  <a class="btn btn-success" href="/stations/new"> Create New Station</a>
+</div>
```
Comment (mimizotti) [inline app/views/stations/index.erb:5]: Extra space between > and Create. Not sure if that matters

**Review by mimizotti — APPROVED** [review]

File: app/views/conditions/edit.erb (hunk 7 above)
Comment (snayrouz) [reply app/views/conditions/edit.erb:34]: Doesn't work yet but it will

**Pull request merged**